### PR TITLE
Added m to cm and cm to m features in calculator

### PR DESCRIPTION
**Pull request by onlyoneaman**
Added centimeter to meter and reverse features in Calculator

### CODE REVIEW

File: Calculator.py [hunk 1]
```diff
@@ -17,3 +17,11 @@ def average(self, nums):
     nums_sum = self.sum(nums)
     result = nums_sum / l
     return result
+  def mettocm(self, nums):
+    conv = 100
+    result = nums*conv
```
Comment (itaditya) [inline Calculator.py:22]: since `nums` is an array, this code would not work

File: Calculator.py [hunk 2]
```diff
@@ -17,3 +17,11 @@ def average(self, nums):
...
+    return result
+  def mettocm(self, nums):
```
Comment (itaditya) [inline Calculator.py:24]: use `_` to separate words, this would become `m_to_cm`

**Comment by itaditya** [comment]
Hi @Amank1647 as I mentioned in the Issue description this issue is not available for people who are not in Road To Hacktober program. But since you have spent some time on this, I'm letting the PR open so it will count in Hacktoberfest.

Also left some comments on the PR so you learn how you can improve.

**Comment by onlyoneaman** [comment]
> Hi @Amank1647 as I mentioned in the Issue description this issue is not available for people who are not in Road To Hacktober program. But since you have spent some time on this, I'm letting the PR open so it will count in Hacktoberfest.
> 
> Also left some comments on the PR so you learn how you can improve.

I worked upon that and improved the program much and I am opening another PR #12 with just m-cm Feature as PR #11 also focusses on cm-m Feature.
Have a look at my PR #12.
Thanks for letting me contribute here  I didn't know earlier that it's for only your Road to Hacktoberfest program and thus I worked upon it. Review and Merge my PR if it's alright so that my work doesn't get wasted.
Thanks Again!!

**Pull request closed without merging**